auth-api/src/main/java/io/stargate/auth/api/AuthApiActivator.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright The Stargate Authors
```

Comment:
Do we use this header in all sub-modules or just in `stargate/stargate`?

Reply:
Do we want the copyright header in `pom.xml` files?

Reply:
All repos